src/components/UserProfile/UserProfile.jsx

```diff
@@ -1,9 +1,135 @@
-import React from 'react'
+import useAuth from '../../contexts/AuthContext';
+import { db } from '../../config/firebase';
+import { doc, getDoc } from '@firebase/firestore';
+import { useEffect, useState } from 'react';
+import styles from './userprofile.module.css';
+import {
+	getStorage,
+	ref,
+	getDownloadURL,
+	uploadBytes,
+} from '@firebase/storage';
+import { toast } from 'react-hot-toast';
 
 const UserProfile = () => {
-  return (
-    <div>UserProfile</div>
-  )
-}
+	const { currentUser } = useAuth();
+	const [user, setUser] = useState();
 
-export default UserProfile
\ No newline at end of file
+	const [age, setAge] = useState();
+
+	const storage = getStorage();
+	const [userProfileImgURL, setUserProfileImgURL] = useState();
+
+	const getUserData = async () => {
```

Comment:
Czy jesteśmy w stanie w tym miejscu zamienić to getUserData (23-29) na to getUserData, które stworzyła Paulina w utils?

Reply:
próbowałam to zmienić, ale wywaliło mi się absolutnie wszystko :D to chyba kwestia tego, że ta funkcja getUserData w UserProfile jest rozbudowana jeszcze o zdjęcie i wiek.